### DISCUSSION TITLE
Claim Expiration, Claim Blocks back to the account

### DISCUSSION THREAD

**Discussion by PirataGu** [GriefPrevention/GriefPrevention#2317] [Ideas]
I guess the title is self explanatory. My request is to add an option to when a player doesnt join the server for more than the configurated ammount of days the Claim Blocks go back to the player instead of just disappearing . Imagine if the player comes back after his claims have been expirated... He would have 0 claim blocks to claim again.

**Comment by Jikoo** [comment]
That's what already happens. Claim usage is subtracted from blocks available. When claims are deleted by any means, they no longer count against the blocks available, and can be used again.

**Comment by PirataGu** [comment]
Does it include extra blocks? (bonus)

**Reply by Jikoo** [reply]
Those are part of the available block total, yes.

**Reply by PirataGu** [reply]
> Those are part of the available block total, yes.

I had a feeling that it didnt give back for some reason... I just test it with a player and everything is working fine. Sorry for the time
